app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -571,6 +575,8 @@ function cellRenderer(params: ICellRendererParams) {
     const valueType = params.value?.type
     if (valueType === 'Float')
       return `<span style="color:grey; font-style: italic;">${params.value?.value ?? 'Unknown'}</span>`
+    else if (valueType === 'Date' || valueType === 'Date_Time' || valueType === 'Time_Of_Day')
+      return formatDateLikeValue(params.value)
```

Comment:
should be ok to use the `_display_text_` method as it will be added to the object.